### ISSUE TITLE
Is this no longer being supported? 

### ISSUE THREAD

**Issue by whiwo** [pwa-builder/pwabuilder-blog#45]
I've created the manifest and added categories and shortcuts (which has worked well in the past) but after generating icons, they only show up in the manifest after I manually add the categories or shortcuts but don't download once the package is generated. 

Instead, it's pulls my primary icon (seen my logo for this post) from my website that's not suitable for a maskable icon. Hence the reason I use a square with just the PF in the middle so it fills whatever shape the device uses. Why is it even pulling items from my site when they're going up into the images folder in my root? This makes no sense. 

I created my own icons, added them to the images folder for upload and fixed the URLs and names so they match in the manifest. 

I consistently get no manifest found, no start_url, no maskable icon and no service worker despite using the advanced caching option. I did notice that your version of Google's workbox is outdated but that shouldn't make a difference. I also noticed your background sync service worker option appears to be the incorrect code. 

Please let me know if I and others are wasting our time if this is no longer supported by Microsoft. I don't really understand why this is so difficult. Putting together a manifest, selecting the service worker, uploading them to the root along with the images (that your system no longer creates for download) and adding the code to my header is no big deal. It's worked in the past but stopped and I've not been able to recreate it ever since. 

The outrageously priced app now being offered in Wordpress for the PWA is a ripoff of epic proportions with poor support (even when paid) and I don't see any other options, If I understood the basics of creating this myself and using Workbox, I'd do it myself. 

In short, please let people know if you're no longer supporting this venture so we an make other arrangements instead of going down a rabbit hole trying to fix a system that's clearly broken. Thanks.
